examples/fluids/problems/newtonian.c

```diff
@@ -137,8 +137,8 @@ PetscErrorCode NS_NEWTONIAN_IG(ProblemData *problem, DM dm, void *ctx) {
                     NULL);
   // -- Conservative vs Primitive variables
   ierr = PetscOptionsBool("-primitive", "Use primitive variables",
-                          NULL, prim_var=PETSC_FALSE, &prim_var, NULL); CHKERRQ(ierr);
-  if (prim_var) {
+                          NULL, use_primitive=PETSC_FALSE, &use_primitive, NULL); CHKERRQ(ierr);
+  if (use_primitive) {
```

Comment:
Thinking ahead, should this be `-state_var [conservative|primitive|entropy]`? It'll be easy enough to change later, but independent flags can conflict, as in `-primitive -entropy`.

Reply:
It should. I was kinda pushing that off until we get a more permanent place for the fluids code (be it ratel or somewhere else).

Reply:
Moved to #1052, as it ended up being a bit involved that I had anticipated (mainly multiple uses of the `SetupContext` structs).